src/operations.py

```diff
@@ -189,9 +189,9 @@ def shift_vertically(item, xdata, ydata, left_scale, right_scale, items):
         ymin = min(x for x in previous_ydata if x != 0)
```

Comment:
If item_ with the trailing underscore is used in the other methods, maybe it makes sense to do this here as well. Unless there's a reason for changing to that elsewhere that I missed.